doc/releases/release-notes-1.14.rst

```diff
@@ -149,6 +149,31 @@ Bluetooth
 * TBD
 * Done several fixes for big endian architectures
 
+* New BLE split software Controller (experimental):
+
+  - Split design with Upper Link Layer and Lower Link Layer
+  - Enabled with CONFIG_BT_LL_SW_SPLIT (disabled by default)
```

Comment:
One option is to add link here to the option documentation, so this could be added
```
:option:`CONFIG_BT_LL_SW_SPLIT`
```
here.

Reply:
fixed